### ISSUE TITLE
Land experimental responsiveness metric in user flows

### ISSUE THREAD

**Issue by brendankenny** [GoogleChrome/lighthouse#13916]
The new [responsiveness metric](https://web.dev/responsiveness/) has mostly settled and so we want to include support for it in situations where there can be (real or automated) user input.

The steps foreseen:

- [ ] add the high-percentile responsiveness metric for timespans
- [ ] add a metric N/A state (that's _not_ `notApplicable`) in the report perf category renderer for when there was no user input in the trace
- [ ] add to navigations if interactions present in the trace, but as a diagnostic audit, not a metric
- [ ] add an `interactions` audit for debugging responsiveness, e.g. here are the keypresses with the slowest response, here's the breakdown of what was blocking the response (time in script, layout, paint, etc)
- [ ] timespans don't yet support simulated throttling, but navigations do. Investigate if simple CPU multipliers are enough to get simulated numbers or if more sophisticated lantern simulations are needed

**Comment by adamraine** [comment]
> add a metric N/A state (that's not notApplicable) in the report perf category renderer for when there was no user input in the trace

What's wrong with using `notApplicable`? Seems like the perfect state for the audit to be in if there was no input.

**Comment by BioPhoton** [comment]
I am extremely hyped about it! Just today I mentioned it to @adamraine.

What you should definitely consider is adding this metric to berformance budgets. 

Would that be possible?

**Comment by adamraine** [comment]
> Would that be possible?

It should be possible as part of #13898

**Comment by BioPhoton** [comment]
Bright future! 🌞

**Comment by brendankenny** [comment]
> > add a metric N/A state (that's not notApplicable) in the report perf category renderer for when there was no user input in the trace
> 
> What's wrong with using `notApplicable`? Seems like the perfect state for the audit to be in if there was no input.

Yeah, I overstated the decision there. Semantically there's definitely an argument there; the biggest drawback is it closes the door to using n/a for what we do for audits everywhere else in the report (remove it from the report), which is yet another special case for metrics/perf category. It still might be the best choice vs signaling within metric details or whatever.

**Comment by adamraine** [comment]
> the biggest drawback is it closes the door to using n/a for what we do for audits everywhere else in the report (remove it from the report)

We don't remove metrics if they are N/A though. This is what a N/A metric looks like right now:

<img width="658" alt="Screen Shot 2022-04-27 at 1 30 04 PM" src="https://user-images.githubusercontent.com/6752989/165625600-414b47d3-1a0e-4c3c-9cd7-91dffffe1604.png">

It seems like we are missing a proper N/A state for metrics, and this would be a good opportunity to fill that void.

**Comment by brendankenny** [comment]
> We don't remove metrics if they are N/A though

yes, but only because there's never been a notApplicable metric before and the display code is `valueEl.textContent = audit.result.displayValue || ''`. There was not a plan here :)

I think the argument is convincing, I just want to make sure to explore the options because it does remove basically the only way we have for an audit to remove itself from the report. If that's something we ever want to do for metrics, we'll either have to change this again (which may not be possible) or add yet another mechanism to do so.
